hydra/definitions.py

```diff
@@ -233,6 +235,26 @@ def NAF(x):
         Gx=0,
         Gy=0,
     ),
+    BANDERSNATCH_ID: Curve(
+        cairo_zero_namespace_name="bandersnatch",
+        id=BANDERSNATCH_ID,
+        p=0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001,
+        n=0x1cfb69d4ca675f520cce760202687600ff8f87007419047174fd06b52876e7e1,
+        h=4,
+        irreducible_polys={},
+        nr_a0=None,
+        nr_a1=None,
+        a=0x73EDA753299D7D483339D80809A1D80553BDA402FFFE5BFEFFFFFFFE1FB22001,  # See https://neuromancer.sk/std/other/Curve25519 for weirstrass form conversion utilities
+        b=0x73EDA753299D7D483339D80809A1D80553BDA402FFFE5BFEFFFFB870D2E00001,
+        b20=None,
+        b21=None,
+        loop_counter=None,
+        line_function_sparsity=None,
+        final_exp_cofactor=None,
+        fp_generator=3, # todo: check if this is true
```

Comment:
use the function is_generator in this file to check. If not, try 2,3,4,5,6,7, etc... you will find one